example_relay.conf

```diff
@@ -19,7 +19,13 @@ max_concurrent: 8
 # Environment variable: $RELAY_DYNAMIC_CONFIG_ROOT
 # Default: none
 # Required: No
-# dynamic_config_root: /tmp/bundle_configs
+# dynamic_config_root: /tmp/dynamic_configs
+
+# Use managed dynamic configuration
+# Requires dynamic_config_root
+# Environment variable: RELAY_MANAGED_DYNAMIC_CONFIG
```

Comment:
We'll need to add this to the doc site: http://docs.operable.io/docs/relay-environment-variables